x/concentrated-liquidity/bench_test.go

```diff
@@ -0,0 +1,235 @@
+				// Normalize lowerTick to be a multiple of tickSpacing
+				lowerTick = lowerTick + (tickSpacing - lowerTick%tickSpacing)
+				// lowerTick <= upperTick <= maxTick
+				upperTick = types.MaxTick - rand.Int63n(int64(math.Abs(float64(types.MaxTick-lowerTick))))
+			}
+			// Normalize upperTick to be a multiple of tickSpacing
+			upperTick = upperTick - upperTick%tickSpacing
+
+			tokenDesired0 := sdk.NewCoin(denom0, sdk.NewInt(rand.Int63n(maxAmountDeposited)))
+			tokenDesired1 := sdk.NewCoin(denom1, sdk.NewInt(rand.Int63n(maxAmountDeposited)))
+
+			accountIndex := rand.Intn(len(s.TestAccs))
+
+			s.createPosition(accountIndex, poolId, tokenDesired0, tokenDesired1, lowerTick, upperTick)
+		}
+
+		// Setup numberOfPositions full range positions for deeper liquidity.
+		if shouldCreateFullRangePositions {
+			for i := 0; i < numberOfPositions; i++ {
+				lowerTick := types.MinTick
+				upperTick := types.MaxTick
+
+				tokenDesired0 := sdk.NewCoin(denom0, sdk.NewInt(maxAmountDeposited).MulRaw(5))
+				tokenDesired1 := sdk.NewCoin(denom1, sdk.NewInt(maxAmountDeposited).MulRaw(5))
+				tokensDesired := sdk.NewCoins(tokenDesired0, tokenDesired1)
+
+				accountIndex := rand.Intn(len(s.TestAccs))
+
+				account := s.TestAccs[accountIndex]
+
+				simapp.FundAccount(s.App.BankKeeper, s.Ctx, account, tokensDesired)
+
+				s.createPosition(accountIndex, poolId, tokenDesired0, tokenDesired1, lowerTick, upperTick)
+			}
+		}
+
+		// Setup numberOfPositions * 2 positions at random ranges around the current tick for deeper
+		// liquidity.
+		if shouldConcentrate {
+			// Within 10 ticks of the current
+			for i := 0; i < numberOfPositions; i++ {
+				lowerTick := currentTick.Int64() - 10
+				upperTick := currentTick.Int64() + 10
+
+				tokenDesired0 := sdk.NewCoin(denom0, sdk.NewInt(maxAmountDeposited).MulRaw(5))
+				tokenDesired1 := sdk.NewCoin(denom1, sdk.NewInt(maxAmountDeposited).MulRaw(5))
+				tokensDesired := sdk.NewCoins(tokenDesired0, tokenDesired1)
+
+				accountIndex := rand.Intn(len(s.TestAccs))
+
+				account := s.TestAccs[accountIndex]
+
+				simapp.FundAccount(s.App.BankKeeper, s.Ctx, account, tokensDesired)
+
+				s.createPosition(accountIndex, poolId, tokenDesired0, tokenDesired1, lowerTick, upperTick)
+			}
+
+			// Within 100 ticks of the current
+			for i := 0; i < numberOfPositions; i++ {
+				lowerTick := currentTick.Int64() - 100
+				upperTick := currentTick.Int64() + 100
+
+				tokenDesired0 := sdk.NewCoin(denom0, sdk.NewInt(maxAmountDeposited).MulRaw(5))
+				tokenDesired1 := sdk.NewCoin(denom1, sdk.NewInt(maxAmountDeposited).MulRaw(5))
+				tokensDesired := sdk.NewCoins(tokenDesired0, tokenDesired1)
+
+				accountIndex := rand.Intn(len(s.TestAccs))
+
+				account := s.TestAccs[accountIndex]
+
+				simapp.FundAccount(s.App.BankKeeper, s.Ctx, account, tokensDesired)
+
+				s.createPosition(accountIndex, poolId, tokenDesired0, tokenDesired1, lowerTick, upperTick)
```

Comment:
Do we not need to normalize the lower tick and upper tick based on tick spacing here?